pystac/catalog.py

```diff
@@ -196,6 +198,13 @@ def __init__(
     def __repr__(self) -> str:
         return "<Catalog id={}>".format(self.id)
 
+    def _repr_html_(self) -> str:
+        if jinja_env:
+            template = jinja_env.get_template("Catalog.jinja2")
+            return str(template.render(catalog=self, catalog_type="Catalog"))
```

Comment:
I think we should rename the `catalog_type` parameter here to something else. "catalog type" in other parts of the code base refers to the values defined in the [`CatalogType` enum](https://github.com/stac-utils/pystac/blob/dfc1cc2102cdfc06b952104f9b2c801e5ee17c3b/pystac/catalog.py#L44). I would also be open to just having separate templates to make it a little more clear how the templates map to the PySTAC classes.

Reply:
I agree, I will go with separating the templates